index.html

```diff
@@ -609,22 +609,22 @@ <h2>DNS-Based Service Discovery</h2>
                         <tr class="rfc2119-table-assertion" id="introduction-dns-sd-service-name">
                             <td><code>_wot._tcp</code></td>
                             <td>Thing</td>
-                            <td>HTTP</td>
+                            <td>HTTP, HTTPS, CoAP over TCP or CoAP over TLS</td>
                         </tr>
                         <tr class="rfc2119-table-assertion" id="introduction-dns-sd-service-name-directory">
                             <td><code>_directory._sub._wot._tcp</code></td>
                             <td>TDD</td>
-                            <td>HTTP</td>
+                            <td>HTTP, HTTPS, CoAP over TCP or CoAP over TLS</td>
                         </tr>
-                        <tr class="rfc2119-table-assertion" id="introduction-dns-sd-service-name-coap">
+                        <tr class="rfc2119-table-assertion" id="introduction-dns-sd-service-name-udp">
                             <td><code>_wot._udp</code></td>
                             <td>Thing</td>
-                            <td>CoAP</td>
+                            <td>CoAP over UDP or CoAP over DTLS</td>
                         </tr>
-                        <tr class="rfc2119-table-assertion" id="introduction-dns-sd-service-name-directory-coap">
+                        <tr class="rfc2119-table-assertion" id="introduction-dns-sd-service-name-directory-udp">
                             <td><code>_directory._sub._wot._udp</code></td>
                             <td>TDD</td>
-                            <td>CoAP</td>
+                            <td>CoAP over UDP or CoAP over DTLS</td>
```

Comment:
Actually, we can include this for completeness but it is a weird option since the directory api is only HTTP at the moment.  But OK, for now.  May have to take it out later if there are no implementations, though.
```suggestion
                            <td>CoAP over UDP or CoAP over DTLS/UDP</td>
```